content/klc/S4.1.adoc

```diff
@@ -2,10 +2,10 @@
 brief = "General pin requirements"
 +++
 
-. Using a `100mil` (`2.54mm`) grid, pin origin must lie on grid node (IEC-60617)
+. Using a `100mil` (`2.54mm`) grid, pin origin must lie on a grid node (IEC-60617)
 . Pins should have a length of at least `100mil` (`2.54mm`)
-* Shorter pins may be allowed to accommodate simple symbols for discrete devices such as resistors, capacitors, etc.
-. Pin length must not exceed `300mil` (`7.62mm`)
-. Pin length can be incremented in steps of `50mil` (`1.27mm`) as required, whilst meeting the absolute length limits as specified above
+* Pin length can be increased in steps of `50mil` (`1.27mm`)
+* Lengthing pins is done based on the number of characters in the pin number. For two characters in the pin number then 100mil long pins should be used, for three characters then 150mil long pins should be used, etc.
```

Comment:
Does this mean that pin 99 should be shorter than pin 100? Or is the idea to base on the longest pin number in the part?

Reply:
Good catch. Added clause to avoid that possibility and minor re-wording for readability.